packages/entities/entities-plugins-metadata/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@kong-ui-public/entities-plugins-metadata",
+  "version": "0.0.1",
```

Comment:
suggestion: since the packaged code has already been in use for a while can we bump the initial version to 1.x?

Reply:
Sure, updated.